packages/ui/src/working-groups/components/WorkingGroupImage.tsx

```diff
@@ -11,59 +11,62 @@ export interface WorkingGroupImageProps {
 
 export const WorkingGroupImage = ({ className, groupName }: WorkingGroupImageProps) => {
   switch (groupName) {
-    case 'Bandwidth':
+    case 'Distribution':
       return (
         <WorkingGroupImageTag viewBox="0 0 108 108" fill="none" className={className}>
-          <rect x="42.8284" y="42.8281" width="22" height="22" fill={Colors.Black[500]} />
-          <circle cx="54" cy="54" r="2" fill={Colors.White} />
-          <path
-            fillRule="evenodd"
-            clipRule="evenodd"
-            d="M53.8284 24L60.2426 30.4142L57.4142 33.2426L55.8284 31.6569V47.8284H51.8284V31.6569L50.2426 33.2426L47.4142 30.4142L53.8284 24Z"
-            fill={Colors.Blue[500]}
-          />
-          <path
-            fillRule="evenodd"
-            clipRule="evenodd"
-            d="M55.8284 42.8281V47.8281H51.8284V42.8281H55.8284Z"
-            fill={Colors.Black[900]}
-          />
-          <path
-            fillRule="evenodd"
-            clipRule="evenodd"
-            d="M53.8284 83.6572L60.2426 77.243L57.4142 74.4146L55.8284 76.0004V59.8288H51.8284V76.0004L50.2426 74.4146L47.4142 77.243L53.8284 83.6572Z"
-            fill={Colors.Blue[500]}
-          />
-          <path
-            fillRule="evenodd"
-            clipRule="evenodd"
-            d="M83.6567 53.8283L77.2425 60.2425L74.4141 57.4141L75.9999 55.8283L59.8283 55.8283L59.8283 51.8283L75.9999 51.8283L74.4141 50.2425L77.2425 47.4141L83.6567 53.8283Z"
-            fill={Colors.Blue[500]}
-          />
-          <path
-            fillRule="evenodd"
-            clipRule="evenodd"
-            d="M24 53.8283L30.4142 60.2425L33.2426 57.4141L31.6569 55.8283L47.8284 55.8283L47.8284 51.8283L31.6569 51.8283L33.2426 50.2425L30.4142 47.4141L24 53.8283Z"
-            fill={Colors.Blue[500]}
-          />
-          <path
-            fillRule="evenodd"
-            clipRule="evenodd"
-            d="M55.8284 59.8281V64.8281H51.8284V59.8281H55.8284Z"
-            fill={Colors.Black[900]}
-          />
-          <path
-            fillRule="evenodd"
-            clipRule="evenodd"
-            d="M47.8284 55.8281L42.8284 55.8281L42.8284 51.8281L47.8284 51.8281L47.8284 55.8281Z"
-            fill={Colors.Black[900]}
-          />
-          <path
-            fillRule="evenodd"
-            clipRule="evenodd"
-            d="M64.8284 55.8281L59.8284 55.8281L59.8284 51.8281L64.8284 51.8281L64.8284 55.8281Z"
-            fill={Colors.Black[900]}
-          />
+          <svg width="109" height="108" viewBox="0 0 109 108" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Since `WorkingGroupImageTag` is already an `svg` currently it renders an `svg` in another `svg`. Please put the svg code (i.e `rect`, `circle`, `path`...) directly as children of `WorkingGroupImageTag` (like it was before). If needed you can pass the width and height props to `WorkingGroupImageTag`